_posts/2016-08-09-collections-app.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Collecting Connections through the Powerbox"
```

Comment:
We tend to use subject case, not title case, in blog posts, based on me scrolling through https://sandstorm.io/news/ , so I guess I suggest lower-casing "C"onnections and "P"owerbox